Documentation/ColumnsConfig/Type/Group/_Properties/_FieldControl.rst.txt

```diff
@@ -88,6 +88,17 @@
         button to open an element browser. It is enabled by default if rendering a
         group element.
 
+        ..  confval:: rendertype
```

Comment:
We are talking about the normal render type here? It does not belong into field control...

Reply:
It does, because I'm using it. See https://github.com/TYPO3/typo3/blob/c4ce32bb04240aa409d0374b3e43e789da7280d2/typo3/sysext/backend/Classes/Form/Element/GroupElement.php#L49

Reply:
Wouldnt the path be:
```
$GLOBALS['TCA'][$table]['columns'][$field]['config']['fieldControl']['elementBrowser']['rendertype']
```

Reply:
you are right, copy & paste mistake. Should I add the other rendertypes as well?

Reply:
What other rendertypes?

Reply:
for insertClipboard, editPopup, ...

Reply:
Yes please